source/testcommon/setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name="opengeh-testcommon",
+    version="0.0.1",
+    description="Shared testing utilities for OpenGEH Python packages",
+    long_description="",
+    long_description_content_type="text/markdown",
+    license="MIT",
+    packages=find_packages(),
+    install_requires=[
+        "pyspark>=3.5.0",
+        "pytest",
+        "PyYAML",
+        "pytest",
+        "delta-spark",
+        "pandas>=1.0.5,<2.0.0",
+        "numpy==1.26.4",
+        "PyArrow >= 4.0.0",
```

Comment:
Are these 3 used?

Reply:
These are related to the dependency issues with `pyspark.testing` that I mention in `testcommon.etl.__init__`. I've added a comment in `setup.py` to ensure that people know why they are there. 

There can be removed in future versions of pyspark, as the pandas integration is being deprecated. But as I could see, this is on the roadmap for spark 4.0.0, so it might be a while...